code/addons/a11y/template/stories/parameters.stories.ts

```diff
@@ -0,0 +1,63 @@
+import globalThis from 'global';
+
+export default {
+  component: globalThis.Components.Button,
+  args: {
+    children: 'Click Me!',
+  },
+  parameters: {
+    chromatic: { disable: true },
+  },
+};
+
+export const Basic = {};
+
+export const Violation = {
+  args: {
+    // empty on purpose to get a button with no text
+    children: '',
+  },
+};
+
+export const Checks = {
+  parameters: {
+    a11y: {
+      config: {},
+      options: {
+        checks: {
+          'color-contrast': { options: { noScroll: true } },
+        },
+        restoreScroll: true,
+      },
+    },
+  },
+};
+
+export const Targetted = {
+  parameters: {
+    a11y: {
+      element: 'button',
+    },
+  },
+};
```

Comment:
Is it obvious this is doing something? Would it be better to do these kind of things in the HTML-based story file?